changelog.json

```diff
@@ -1,5 +1,25 @@
 {
   "releases": {
+    "1.1.1-beta2": [
+      "[New] Render WebP images in diffs - #4164. Thanks @agisilaos!",
+      "[Fixed] Edit context menus in commit form input elements - #3886",
+      "[Fixed] Escape behavior should match that of Branch List - #3972",
+      "[Fixed] Caret position always goes to end after inserting completion - #3835. Thanks @CarlRosell!",
+      "[Fixed] Handle error events when watching files used to get Git LFS output - #4117",
+      "[Fixed] Potential race condition when opening a fork pull request - #4149",
```

Comment:
is this user-facing? feel free to push back

Reply:
I believe so. Before #4149 the user would select the fork branch, but the checkout would silently fail because the `git remote add` wasn't done, leaving them on their original branch without any indicator that we failed to checkout successfully.